lib.php

```diff
@@ -34,3 +34,15 @@ function block_configurable_reports_security_checks() {
     return [new block_configurable_reports\check\sql_execution()];
 }
 
+function block_configurable_reports_pluginfile($course, $birecord, $context, $filearea, $args, $forcedownload, $sendfileoptions) {
+    global $CFG;
+    if ($filearea !== 'export') {
+        send_file_not_found();
+    }
+    $exporttype = $args[0];
+    if (!in_array($exporttype, ['csv','json', 'ods', 'xls']) ){
+        send_file_not_found();
+    }
+    $filename = $CFG->dirroot.'/blocks/configurable_reports/'.$filearea.'/'.$exporttype.'/pix.gif';
```

Comment:
This one doesn't feel right, shouldn't this just be normal pix serving which ends up via /theme/image.php/xxx ?

Reply:
Hey @brendanheywood - I tried using theme url, but that doesn't work in pluginfile. 
In UI, the image URL is showing like it is coming from pluginfile. I hope this is fine.
![image](https://user-images.githubusercontent.com/5063130/150463718-e23bbf02-736a-44ce-acb4-3b86818d201a.png)